fuel/app/classes/materia/widget/asset.php

```diff
@@ -271,6 +273,13 @@ public function render(string $size): void
 			header("Content-Length: {$bytes}");
 			header('Content-Transfer-Encoding: binary');
 			header('Cache-Control: max-age=31536000');
+
+			// Special case for 3D labeling
```

Comment:
A little more context might be necessary here.

Reply:
Updated comment with additional context.